packages/leva/src/components/Button/Button.tsx

```diff
@@ -5,7 +5,7 @@ import { Row } from '../UI'
 import { StyledButton } from './StyledButton'
 
 type ButtonProps = {
-  label: string
+  label: string | JSX.Element
```

Comment:
I'm not sure this makes sense. I can imagine a button having the ability to be triggered by a key, then adding a label to it, then adding buttons to the joystick and having them trigger changes in the joystick. 
Except right here I just added labels.
Maybe they shouldn't even be buttons. I used buttons because they were readily available and thought on a touch device they would be nice, but I just tested them and they are very hard to hit below the joystick and would make sense to the left or right instead. Thoughts vs feature creep?